scripts/extras/foglamp.sudoers

```diff
@@ -1 +1 @@
-%sudo ALL=(ALL) NOPASSWD: /usr/bin/apt-get -y update, /usr/bin/apt-get -y install foglamp, /usr/bin/apt -y install /usr/local/foglamp/data/plugins/foglamp*.deb, /usr/bin/apt list, /usr/bin/apt -y install foglamp*
+%sudo ALL=(ALL) NOPASSWD: /usr/bin/apt update, /usr/bin/apt-get -y install foglamp, /usr/bin/apt -y install /usr/local/foglamp/data/plugins/foglamp*.deb, /usr/bin/apt list, /usr/bin/apt -y install foglamp*, /usr/bin/apy -y upgrade
```

Comment:
~apy~ apt

Reply:
Thank you

Reply:
@MarkRiddoch if we are replacing the command with `apt` instead of `apt-get` on update then we should consider https://github.com/foglamp/FogLAMP/blob/develop/scripts/extras/update_task.apt#L57 file changes

Reply:
Fixed in https://github.com/foglamp/FogLAMP/pull/1681/commits/536aeaee798b4bcb08cf2ab1cb00d4c0a4f304e8